suites/quincy/rbd/tier-2_rbd_regression.yaml

```diff
@@ -143,6 +143,12 @@ tests:
       name: Test snap and clone creation on restored image from trash
       polarion-id: CEPH-11413
 
+  - test:
+      desc: Verify parent-clone relationship and clone IOs are not affected when parent moved to trash
+      module: rbd_clone_trash_restore_parent_image.py
```

Comment:
same as above.

Reply:
Done. Please let me know if this new name looks okay.